infra/main.bicep

```diff
@@ -104,6 +104,9 @@ param azureOpenAIModelName string = 'gpt-35-turbo'
 
 param azureOpenAIModelVersion string = '0613'
 
+@description('Azure OpenAI Model Capacity')
+param azureOpenAIModelCapacity int = 30
```

Comment:
Can we add a one liner explaining what this is?

Reply:
One liner where? A code comment or in the description? Feels like it may be better suited to a configuration page/doc.

Maybe we can just link to this page https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/quota ? WDYT?